docs/conda.rst

```diff
@@ -0,0 +1,275 @@
+.. inclusion-marker-start-do-not-remove
+
+Building a Conda environment with GPU support for Horovod
+=========================================================
+
+In this section we describe how to build Conda environments for my deep learning projects using 
+Horovod to enable distributed training across multiple GPUs (either on the same node or 
+spread across multuple nodes).
+
+Installing the NVIDIA CUDA Toolkit
+----------------------------------
+
+First thing you need to do is to install the `appropriate version`_ of the NVIDIA CUDA Toolkit on 
```

Comment:
First thing you need to do is to install --> Install

Reply:
Why capitalize Install in this context?

Reply:
I think she means drop the "First thing you need to do is..." and just say "Install the appropriate version..."

Reply:
Done